code/parse/sexp.cpp

```diff
@@ -5094,6 +5094,11 @@ int rand_sexp(int node, bool multiple)
 	if (is_nan_forever)
 		return SEXP_NAN_FOREVER;
 
+	if (low > high) {
+		Warning(LOCATION, "rand%s was passed an invalid range (%d ... %d)!", multiple ? "-multiple" : "", low, high);
+		return SEXP_KNOWN_FALSE;
```

Comment:
This should return `low`, both for compatibility reasons (it's what used to be returned) and because caller sexps expect a number, not a boolean.